lib/bowling_cases.rb

```diff
@@ -0,0 +1,47 @@
+class BowlingCase < OpenStruct
+  def test_name
+    'test_%s' % description.downcase.tr(' ', '_')
+  end
+
+  def skipped
+    index.zero? ? '# skip' : 'skip'
+  end
+
+  def work_load
+    if assert_error?
```

Comment:
Oops, the `assert_error?` conditional check isn't required in `assert`.

If we moved the logic to the `assert` method, we will still have one `assert_error?` check?